tests/appsec/iast/source/test_multipart.py

```diff
@@ -0,0 +1,39 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import coverage, weblog, bug
+from .._test_iast_fixtures import SourceFixture
+
+
+@coverage.basic
+class TestMultipart:
+    """Verify that multipart parameter is tainted"""
+
+    source_fixture = SourceFixture(
+        http_method="POST",
+        endpoint="/iast/source/multipart/test",
+        request_kwargs={"files": {"file1": ("file1", "bsldhkuqwgervf")}},
+        source_type="http.request.multipart.parameter",
+        source_name="Content-Disposition",
+        source_value=None,
+    )
+
+    def test_source_reported(self):
+        self.source_fixture.test()
+
+    @bug(library="java", reason="Not working as expected")
```

Comment:
Is there a jira ticket for this? if yes, you can simply set the JIRA ticket as `reason`

Reply:
It's a common issue across most telemetry tests in this directory, we'll do an overall review some time soon.

Reply:
Oh, actually it doesn't apply to this test.